[2023-11-14T10:56:14.398+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: extract_dag.extract_task manual__2023-11-14T10:56:11.865660+00:00 [queued]>
[2023-11-14T10:56:14.404+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: extract_dag.extract_task manual__2023-11-14T10:56:11.865660+00:00 [queued]>
[2023-11-14T10:56:14.404+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-11-14T10:56:14.404+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-11-14T10:56:14.404+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-11-14T10:56:14.448+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): extract_task> on 2023-11-14 10:56:11.865660+00:00
[2023-11-14T10:56:14.451+0000] {standard_task_runner.py:55} INFO - Started process 12577 to run task
[2023-11-14T10:56:14.455+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'extract_dag', 'extract_task', 'manual__2023-11-14T10:56:11.865660+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/extract_dag.py', '--cfg-path', '/tmp/tmp3d2mcp9i']
[2023-11-14T10:56:14.456+0000] {standard_task_runner.py:83} INFO - Job 3: Subtask extract_task
[2023-11-14T10:56:14.520+0000] {task_command.py:388} INFO - Running <TaskInstance: extract_dag.extract_task manual__2023-11-14T10:56:11.865660+00:00 [running]> on host codespaces-1a8ad4
[2023-11-14T10:56:14.638+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=extract_dag
AIRFLOW_CTX_TASK_ID=extract_task
AIRFLOW_CTX_EXECUTION_DATE=2023-11-14T10:56:11.865660+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-14T10:56:11.865660+00:00
[2023-11-14T10:56:14.639+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-11-14T10:56:14.639+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'wget -c https://datahub.io/core/top-level-domain-names/r/top-level-domain-names.csv.csv -O         workspaces/hands-on-introduction-data-engiineering-4395021/lab/orchestrated/airflow-extract-data.csv']
[2023-11-14T10:56:14.646+0000] {subprocess.py:86} INFO - Output:
[2023-11-14T10:56:14.650+0000] {subprocess.py:93} INFO - workspaces/hands-on-introduction-data-engiineering-4395021/lab/orchestrated/airflow-extract-data.csv: No such file or directory
[2023-11-14T10:56:14.651+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-11-14T10:56:14.657+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/python/3.10.8/lib/python3.10/site-packages/airflow/operators/bash.py", line 196, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-11-14T10:56:14.659+0000] {taskinstance.py:1318} INFO - Marking task as FAILED. dag_id=extract_dag, task_id=extract_task, execution_date=20231114T105611, start_date=20231114T105614, end_date=20231114T105614
[2023-11-14T10:56:14.703+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 3 for task extract_task (Bash command failed. The command returned a non-zero exit code 1.; 12577)
[2023-11-14T10:56:14.709+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-11-14T10:56:14.718+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
